gpt4all-backend/CMakeLists.txt

```diff
@@ -71,18 +73,32 @@ foreach(BUILD_VARIANT IN LISTS BUILD_VARIANTS)
                      PROPERTY INTERPROCEDURAL_OPTIMIZATION ${IPO_SUPPORTED})
     endfunction()
 
-    # Add each individual implementation
-    add_library(llamamodel-${BUILD_VARIANT} SHARED
+    # Add each individual implementations
```

Comment:
nitpick, you don't want the plural here

Reply:
I noticed that as well, but decided to leave it as is since it's not worth a commit. Will batch this with further things that may come up.